### DISCUSSION TITLE
Catalan Language Leader

### DISCUSSION THREAD

**Discussion by synesthesiam** [OHF-Voice/intents#340] [Language Leaders]
@duhow, @emitoni, @jorclaret - is anyone interested in being a language leader for Catalan?

**Comment by duhow** [comment]
I'll let others to be first, if nobody shows up then I'm in.
In any case, I'll continue to post updates for Catalan and Spanish as well :)

**Comment by gezquerra** [comment]
Hello, I don't have much experience with Github but I have a lot with Home Assistant. I am at your disposal to collaborate on Catalan intents.

**Comment by synesthesiam** [comment]
@mardito, @ArnyminerZ tagging you here for the leader discussion.

@duhow I will add you as leader for now :+1:

**Reply by ArnyminerZ** [reply]
I can help with some contributions, but not much time to be a proper leader, sorry 😓 If @duhow would like to take the lead, it's perfect for me 😄

**Comment by synesthesiam** [comment]
@duhow @ArnyminerZ - would you or others have time to translate the new responses into Catalan?

**Reply by duhow** [reply]
Ah, sure!

**Reply by gezquerra** [reply]
Here too, count on me.

**Reply by synesthesiam** [reply]
Thanks everyone!

**Comment by synesthesiam** [comment]
@duhow when you get a chance, can you please look into translating the sentences for `HassGetState`?
